vignettes/DataAnalysis.Rmd

```diff
@@ -209,12 +218,12 @@ Data visualizations are built into each `Assess()` function/KRI. For a Poisson A
 
 # Appendix 1 - Supporting Functions {#appendix-1}
 
-The following sections include various examples of supporting functions and statistical models that can be employed in the `Assess()` functions. Please note that this is **not** an exhaustive list, but includes some of the most commonly called upon functions.
+The following sections include various examples of supporting functions and statistical models that can be employed in the Analytics workflow. Please note that this is **not** an exhaustive list, but includes some of the most commonly called upon functions.
 
 ### Mapping Functions
 
--   `Map_Raw()`: Creates input data using rawplus data
--   `Map_Adam()`: Creates input data using CDISC-compliant ADaM data
+-   `RunQuery()`: Run a SQL query to create new data.frames with filtering and column name specifications.
```

Comment:
Is there a convention that leads to this being `RunQuery()` and not `Run_Query()`, while the rest of the functions are `Upper_Snake_Camel_Case()`?

Reply:
Nevermind, I think; the `_` in function names denotes families of functions. Although `Input_Rate()` doesn't really match that pattern, but I guess we could conceivably support other inputs.